components/sst_importer/src/sst_importer.rs

```diff
@@ -230,6 +230,7 @@ impl SstImporter {
         dst_file: std::path::PathBuf,
         backend: &StorageBackend,
         expect_sha256: Option<Vec<u8>>,
+        use_kms: bool,
```

Comment:
maybe `support_kms` is better, since not all download sst requests enable kms feature. but do_download always pass true.

Reply:
We can encrypt kv files with KMS when `backup log` in TiKV. But `restore meta log` can not support kms currently in br client.